pom.xml

```diff
@@ -89,19 +89,11 @@
 			</dependency>
 
 			<!-- End OpenMRS core -->
-			
 			<dependency>
-				<groupId>org.mockito</groupId>
-				<artifactId>mockito-all</artifactId>
-				<version>1.8.5</version>
-				<scope>test</scope>
-			</dependency>
-
-			<dependency>
-				<groupId>org.codehaus.groovy</groupId>
```

Comment:
Is there a reason why you changed it from `org.codehaus.groovy`?

Reply:
Yes, org.codehaus.groovy has been relocated to org.apache.groovy. https://mvnrepository.com/artifact/org.codehaus.groovy/groovy-all

Reply:
Which one are we using in platform 2.7.3?

Reply:
org.codehaus.groovy:2.4.21. Should I update it there or remove it from here?

https://github.com/openmrs/openmrs-core/blob/02babbcd3d63f717d3f1a118e790b5b957dfa11d/api/pom.xml#L244-L248

Reply:
You can update the one in core

Reply:
Can you also update the version?